program/FORCE-RISCV ISG preliminary project proposal.md

```diff
@@ -0,0 +1,145 @@
+- RISCV-DV would be not possible to be extended further in features beyond certain point.
+- Dependency on (customized) GCC toolchain.
+- Needs commercial license for System Verilog compile/simulating tools to contribute.
+- OpenHW Group don't have much influence on the project direction.
+
+## External dependencies
+- Third party open source libraries/projects with BSD/Apache like/compatible license would need to be updated periodically (quarterly or bi-yearly) to keep up with upstream.
+
+## List of project outputs
+- Buildable working source code supporting both 64-bit and 32-bit RISC-V.
+- User manual.
+- Examples and tutorials.
+- Regression system.
+
+## TGs Impacted/Resource requirements
+- This work will be planned and discussed in the Verification TG. The project plan will be produced by the project manager with assistance of Verification TG, and reviewed/approved in TWG.
+
+## OpenHW engineering staff resource plan: requirement and availability
+- Duncan Bees / Mike Thompson - program management oversight
+
+## Engineering resource supplied by members - requirement and availability
+- Futurewei Technologies will lead the engineering effort.
+- We believe this project will benefit OpenHW CPU verification and the ecosystem as a whole. Other members can be involved as follows and increase their involvement as value of the project become more apparent to them:
+  �    Setting requirements and feature requests
+  �    Help with ISG development.
+  �    Help with ISG frontend test template and library development.
+  �    Use FORCE-RISCV in OpenHW Group projects, develop test templates for verification purpose and provide feedback.
+
+## OpenHW marketing resource - requirement and availability
+- No additional resource requirements yet identified
+
+## Marketing resource supplied by members - requirement and availability
+- No  resource requirements yet identified
+
+## Funding supplied by OpenHW - requirement and availability 
+- No funding requirements yet identified
+
+## Funding supplied by members - requirement and availability
+- No funding requirements yet identified
+
+## Architecture diagram
+- Slides presented to TWG on 09/07/2020 contains architecgture diagram: https://github.com/openhwgroup/core-v-docs/blob/master/TWG/MeetingPresentations/2020-09-07%20FORCE-RISCV%20ISG%20-%20status.pdf
+
+## Who would make use of OpenHW output
+CVA6 project will be able to start using FORCE-RISCV 64-bit version.
+OpenHW Group members working on 64-bit and/or 32-bit RISC-V processor designs will directly benefit from the FORCE-RISCV with both 64-bit and 32-bit support in the long term.
+
+## Project license model
+The project code will be using Apache 2.0 license.
+
+## Description of initial code contribution, if required
+Initial code contribution has happened on June 12, 2020 from Futurewei Technologies.

```

Comment:
Add beneath code contribution point,
Contribution Questionnaire under Eclipse Foundation process to be done

Reply:
Good point.